notebooks/261-fast-segment-anything/261-fast-segment-anything.ipynb

```diff
@@ -18,6 +18,8 @@
     "- [Original Pipeline Inference](#FastSAM-in-Ultralytics)\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    for image in tqdm(coco_dataset):
</pre><p>Maybe add description to <code>tqdm</code> like "Measuring inference time"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1413/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
added!